lib/tapioca/dsl/compilers/active_record_relations.rb

```diff
@@ -593,6 +594,33 @@ def create_common_methods(common_relation_methods_module)
             end
           end
 
+          SIGNED_FINDER_METHODS.each do |method_name|
+            case method_name
+            when :find_signed
+              create_common_method(
+                "find_signed",
+                common_relation_methods_module,
+                parameters: [
+                  create_param("signed_id", type: "T.untyped"),
+                  create_kw_opt_param("purpose", type: "T.untyped", default: "nil")
+                ],
+                return_type: as_nilable_type(constant_name),
+              )
+            when :find_signed!
+              create_common_method(
+                "find_signed!",
+                common_relation_methods_module,
+                parameters: [
+                  create_param("signed_id", type: "T.untyped"),
+                  create_kw_opt_param("purpose", type: "T.untyped", default: "nil")
+                ],
+                return_type: constant_name,
+              )
+            else
```

Comment:
Should we add `signed_id_verifier` and `signed_id_verifier=` as well? The nice thing about the `case`/`when` matching is that even if those methods are defined in Rails 7.0 or something, it still works properly in Rails 6.1, since they will never get matched from the actual instance methods on the module.

Reply:
I wasn't entirely sure about that one either, but more about the fact that configuring the verifier has little to do with with AR relations. Since I had no need for setting this let alone better typing for it, I decided to let it be :)